composer.json

```
@@ -23,7 +23,7 @@
     "opis/json-schema": "^2.3"
   },
   "require-dev": {
-    "larastan/larastan": "^2.8",
+    "larastan/larastan": "^2.8 || ^3.0",
```

As the dependency is a dev dependency, there is no need to keep v2. 

```suggestion
    "larastan/larastan": "^3.0",
```